docs/release-notes/dont-show-inactive-users.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Breaking Changes**
+
+-  Users: The CLI command ``det user list`` by default will no longer show inactive users. To list
+   both active and inactive users ``det user list --all`` can be used. The SDK method ``list_users``
+   will also no longer show inactive users unless the paramter ``all_users`` is set to true.
```

Comment:
to fix typo "paramter" and rewrite for clarity:

Users: The CLI command "det user list" will no longer show inactive users by default. To list both active and inactive users, you can use the "det user list --all" command. Similarly, the SDK method "list_users" will no longer display inactive users unless the "all_users" parameter is explicitly set to true.